lwcomponent/cdk_component.go

```diff
@@ -18,21 +18,21 @@ const (
 )
 
 type CDKComponent struct {
-	Name           string
-	Description    string
-	Type           Type
-	Status         Status
-	InstallMessage string
-	UpdateMessage  string
+	Name           string `json:"name"`
+	Description    string `json:"description"`
```

Comment:
This is the JSON:

```
{"name":"component-example","description":"Component description","type":"CLI_COMMAND","apiInfo":{"id":2,"name":"component-example","version":"0.8.2","allVersions":["0.8.0","0.8.1","0.8.2"],"desc":"Component description","sizeKB":0,"deprecated":false,"componentType":"CLI_COMMAND"}}
```

Can see that we are duplicating information, we only need to cache the APIinfo.  Can you remove `name`, `description`, and `type`.

Reply:
Updated.